app/controllers/games_controller.rb

```diff
@@ -0,0 +1,45 @@
+class GamesController < ApplicationController
+
+  def index
+    @games = Game.available
+  end
+
+  def create
+    # For Development Purposes Only - Used to test Available list.
+    Game.create(white_player_id: 1)
```

Comment:
Can we change this to `Game.create(white_player_id: current_user.id)` so that it pulls from the currently logged-in user?